examples/accesssettings/accesssettings.go

```diff
@@ -2,13 +2,14 @@ package main
 
 import (
 	"fmt"
+	"github.com/sendgrid/sendgrid-go"
 	"log"
 	"os"
 )
 
-// Retrieveallrecentaccessattempts : Retrieve all recent access attempts
+// GetRecentAccessAttempts : Retrieve all recent access attempts
```

Comment:
These should match the corresponding string, since they are generated from our [OpenAPI spec](https://github.com/sendgrid/sendgrid-oai). So in this case, it would be `RetrieveAllRecentAccessAttempts`. Same is true for all the strings you have updated.